data/brands/shop/cheese.json

```
@@ -0,0 +1,20 @@
+{
+  "properties": {
+    "path": "brands/shop/cheese",
+  },
+  "items": [
+    {
+      "displayName": "באשר פרומז'רי",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "brand": "Basher Fromagerieil",
```

```suggestion
        "brand": "באשר פרומז'רי",
```
Better to use the more common language for the brand.